packages/paste-website/src/components/DoDont.tsx

```diff
@@ -37,11 +42,10 @@ const StyledExampleImg: React.FC<ExampleProps> = styled.div`
 `;
 
 const Example: React.FC<ExampleProps> = props => {
+  const hasImage = props.image;
   return (
     <div>
-      <StyledExampleImg {...props}>
-        <img src={props.image} alt="" />
-      </StyledExampleImg>
+      <StyledExampleImg {...props}>{hasImage ? <img src={props.image} alt="" /> : null}</StyledExampleImg>
```

Comment:
Is an empty `div` necessary? can you conditionally render the entire block with something like:

```
{hasImage && <StyledExampleImg {...props}><img src={props.image} alt="" /></StyledExampleImg>}
```

Reply:
Mmmmm I see, the image container has the border. Ideally that'd move to the outer example box and then the image container would be entirely optional.

Reply:
Moved those styles to the out box. Also replaced the remaining divs with Box.